### PR TITLE
use html calc method to calculate treemap-title sizes

### PR DESCRIPTION
**Pull request by feitzi**
Should fix the error telerik/kendo-themes#680

Instead of using calculation the values for top and left with scss use the browser calc method.  
This is required if you use a `rem` or `em` font-size value

### CODE REVIEW

**Comment by joneff** [comment]
@feitzi , can you rebase your branch from latest develop and change the name of the commit to `fix(treemap): use html calc method to calculate treemap-title-sizes`. We use commitzen to extract the change log based on the commit messages

File: packages/default/scss/dataviz/_layout.scss [hunk 1]
```diff
@@ -347,10 +347,10 @@ $chart-title-font-size: 1.143em !default;
             right: 0;
         }
         .k-treemap-title + .k-treemap-wrap {
-            top: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
+            top: calc( #{($padding-y-sm*2)} + (#{$line-height}*#{$font-size}) + 1px);
```
Comment (joneff) [inline packages/default/scss/dataviz/_layout.scss]:
```suggestion
            top: calc( #{$padding-y-sm * 2} + (#{$line-height} * #{$font-size}) + 1px);
```

File: packages/default/scss/dataviz/_layout.scss [hunk 2]
```diff
@@ -347,10 +347,10 @@ $chart-title-font-size: 1.143em !default;
...
         }
         .k-treemap-title-vertical + .k-treemap-wrap {
-            left: calc( #{($padding-y-sm*2) + ($line-height*$font-size)} + 1px);
+            left: calc( #{($padding-y-sm*2)} + (#{$line-height}*#{$font-size}) + 1px);
```
Comment (joneff) [inline packages/default/scss/dataviz/_layout.scss]:
```suggestion
            left: calc( #{$padding-y-sm * 2} + (#{$line-height} * #{$font-size}) + 1px);
```

Comment (feitzi) [reply packages/default/scss/dataviz/_layout.scss]: done

File: packages/default/scss/dataviz/_layout.scss (hunk 1 above)
Comment (feitzi) [reply packages/default/scss/dataviz/_layout.scss]: done

**Comment by feitzi** [comment]
I squashed the commits and added your change requests.

**Pull request merged**